library.properties

```diff
@@ -4,6 +4,6 @@ author=Lucas Chiesa <lucas.chiesa@gmail.com>
 maintainer=Ernesto Corbellini <ecorbe@gmail.com>
 sentence=A library that makes using your Dreamster a breeze.
 paragraph=This library supports all the basic functions of the Dreamster robot
-category=Robotics
+category=Device Control
```

Comment:
Because it was giving an error when loading the library. According to the [specification](https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification) it must be one of the listed values.